packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -1168,20 +1179,21 @@ export async function requestOAuth1(
 		secret: oauthTokenData.oauth_token_secret as string,
 	};
 
-	// @ts-ignore
+	// @ts-expect-error @TECH_DEBT: Improve typings for requestOptions
 	requestOptions.data = { ...requestOptions.qs, ...requestOptions.form };
```

Comment:
I'd rather change this to `// @ts-expect-error @TECH_DEBT: get rid of the request library` 😅